### PR TITLE
fix(ejp-xml-pipeline): File write closed before being read

### PR DESCRIPTION
**Pull request by tayowonibi**
using context

Test for this is not yet written (to be written latter)
After deployment, all files, say from beginning of the month needs to be rewritten. I chose the beginning of the month because we've go backdate ejp xml dump during the month.

### CODE REVIEW

File: ejp_xml_pipeline/etl.py [hunk 1]
```diff
@@ -39,11 +39,12 @@ def write_entities_in_parsed_doc_to_file(
 @contextmanager
 def get_opened_temp_file_for_entity_types(
         ejp_xml_data_config: eJPXmlDataConfig,
+        file_dir: str
```
Comment (de-code) [inline ejp_xml_pipeline/etl.py:42]: minor: maybe would keep the same variable name

File: ejp_xml_pipeline/etl.py [hunk 2]
```diff
@@ -39,11 +39,12 @@ def write_entities_in_parsed_doc_to_file(
...
 ):
-    with ExitStack() as stack, TemporaryDirectory() as tempdir:
+    with ExitStack() as stack:
         for ent_db_load_config \
                 in ejp_xml_data_config.entity_type_mapping.values():
-            ent_db_load_config.set_directory(tempdir)
+            ent_db_load_config.set_directory(file_dir)
```
Comment (de-code) [inline ejp_xml_pipeline/etl.py:47]: minor: Not part of this PR, but maybe it would be better to not update the config here. Maybe the directory could be passed to `get_full_file_location`.

**Review by de-code — APPROVED** [review]

**Comment by de-code** [comment]
What was the issue, the temporary directory was deleted before the file was processed?

**Comment by tayowonibi** [comment]
> What was the issue, the temporary directory was deleted before the file was processed?

files opened for write were not closed before being read. so, some files were not read for writing into bigquery
